opwen_email_server/utils/email_parser.py

```diff
@@ -1,4 +1,5 @@
 from base64 import b64encode
+from bs4 import BeautifulSoup
```

Comment:
This is a third party module so please move it next to the other third party imports, i.e. next to pyzmail.